src/recoil/rankerState.ts

```diff
@@ -0,0 +1,44 @@
+import { atom, selector, useRecoilValueLoadable } from 'recoil';
+
+import { requestRankers } from '@request';
+
+const rankingLimit = 10;
+
+export interface IRankerState {
+  ranking: number;
+  nickname: string;
+  stampCount: number;
+  profileUrl: string | null;
+  power: number;
+  email: string;
+}
+
+const defaultRankerState: IRankerState = {
+  ranking: 0,
+  nickname: '',
+  stampCount: 0,
+  profileUrl: null,
+  power: 1,
+  email: '',
+};
+
+export const rankerState = atom<IRankerState>({
+  key: 'rankerState',
+  default: defaultRankerState,
+});
+
+export const rankerAsyncState = selector({
```

Comment:
selector를 모듈 내에 감추고 useRankerRecoilValue로 외부에 노출하는 방법을 쓰고 있기 때문에
굳이 export를 안해주셔도 될것 같습니다.

```suggestion
const rankerAsyncState = selector({
```

Reply:
selector의 반환값이 현재 `state` 보다는 `readonly value` 로 추론되는 것 같습니다.
그리도 비동기 데이터라는 정보는 필요없는 정보처럼 보입니다.
변수명을 바꿔주시는 건 어떨까요??

```suggestion
export const rankerValue = selector({
```